.github/workflows/build.yml

```diff
@@ -78,20 +78,20 @@ jobs:
         username: ${{ github.actor }}
         password: ${{ secrets.GITHUB_TOKEN }}
         registry: docker.pkg.github.com
-        repository: vkuznet/RucioTracers/RucioTracers
+        repository: dmwm/ruciotracers/ruciotracers
         tag_with_ref: true
 
-#     - name: Login to DockerHub
-#       uses: docker/login-action@v1
-#       with:
-#         username: ${{ secrets.DOCKER_HUB_USERNAME }}
-#         password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+    - name: Login to DockerHub
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKER_HUB_USERNAME }}
+        password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
 
-#     - name: Build and push
-#       uses: docker/build-push-action@v2
-#       with:
-#         context: .
-#         file: ./Dockerfile
-#         load: true
-#         tags: cmssw/RucioTracers:${{steps.get-ref.outputs.tag}}
-#     - run: docker push cmssw/RucioTracers:${{steps.get-ref.outputs.tag}}
+    - name: Build and push
+      uses: docker/build-push-action@v2
+      with:
+        context: .
+        file: ./Dockerfile
+        load: true
+        tags: cmssw/ruciotracers:${{steps.get-ref.outputs.tag}}
+    - run: docker push cmssw/ruciotracers:${{steps.get-ref.outputs.tag}}
```

Comment:
docker repo is cmssw/rucio-tracers.

Reply:
it does not matter here you only define the name of the image, if you want it can be cmssw/rucio-tracers.

Since you use so many different combinations, i.e. RucioTracers, rucio-tracers, and github wants lowercase I rather suggest to rename repo to one naming convention, rucio-tracers and use lowercase everywhere.

Reply:
please update the image to cmssw/rucio-tracers.